src/PKSim.Presentation/Presenters/Charts/ChartPresenter.cs

```diff
@@ -208,14 +208,14 @@ protected virtual void AddObservedData(IReadOnlyList<DataRepository> observedDat
          }
       }
 
-      protected virtual void AddColorGroupedObservedData(IReadOnlyList<IReadOnlyList<DataRepository>> observedData)
+      protected virtual void AddColorGroupedObservedData(IReadOnlyList<IReadOnlyList<DataRepository>> observedDataListGroupedByFolder)
       {
          using (_chartUpdater.UpdateTransaction(Chart))
          {
-            foreach (var observedDataNodesList in observedData)
+            foreach (var observesDataListInFolder in observedDataListGroupedByFolder)
```

Comment:
@georgeDaskalakis I have renamed this because we are not dealing with nodes ad this stage and this confused me for a bit